the-collab-lab/tcl-35-smart-shopping-list
# Issue #2 | Add list view and add item view

_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._ 

## Description



This pull request contains implementation code for a user of the smart shopping list app to navigate between the "list" view and "add an item" view. In this PR , Dana and I (Edema) created components for these views and clearly delineated them from each other with appropriate routing to navigate to each of these views ("list view" and "add an item view"). We installed react-router-dom dependency  and used it for routing. One among the things we learnt in this sprint work was how to collaborate effectively and come up with a working UI.

## Related Issue



## Acceptance Criteria

- [x] react-router-dom was added as a project dependency
- [x] Whichever view is selected, the corresponding link is displayed in bold text



## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|   ✓ | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

### Before




### After



## Testing Steps / QA Criteria


* From your terminal, pull down this branch with `git pull origin dc-eu-add-two-views` and check that branch out with `git checkout dc-eu-add-two-views` 
* Then `npm update or npm install` to install the newly added dependencies locally and `npm start` to launch the app.
cypress tests can be added to check for the availability and display of each UI feature and the appropriate texts and components is displayed on each particular routing. Though tests were not prioritised for this particular implementation stage.
